R/utility_spliceHelper_functions.R

```diff
@@ -572,6 +688,28 @@ dropGrangesListElementsByWidth <- function(grangesList, minWidth = 5,
     return(relist(unlistedExons, partitioning))
 }
 
+#' Function that selects the first exon from an IRangesList object
+#' @param range IRangesList with elements required to be ordered by coordinates
+#' @param stand strand
+#' @noRd
+selectStartExonFromRangesList <- function(range, strand){
```

Comment:
similarly for these two functions: `selectStartExonFromGrangesList` and `selectEndExonFromGrangesList`, they look very similar to me, they can potentially be one function and they don't even need to have an extra variable I feel.

Reply:
Hi @cying111, yes they are very similar, and in fact, the first, second, and last line of the functions are the same. I don't think I can merge them into one function because the direction of the third lines of the two functions are different, where for selectStartExonFromRangesList(), the variable from the second line of the function is used while selectEndExonFromRangesList() returns the variable from the first line of the function.

Reply:
I guess I will also keep this one.

Reply:
I see, for this one, I would probably still combine as they are all numeric vectors, so the comparison with if would not slow down the function much, I feel. It would still be good to have less functions that with similar internal designs though.

Reply:
I have combined the two functions.